client/components/CustomBankField.js

```diff
@@ -0,0 +1,66 @@
+import {has, translate as t} from '../Helpers';
+import T from './Translated';
+
+export default class CustomBankField extends React.Component {
+
+    constructor(props) {
+        has(props, 'params');
```

Comment:
There are a lot of other params : type, placeholderKey, placeholder, label, labelKey…
That's why I used an object.